editoast/src/core/conflict_detection.rs

```diff
@@ -75,10 +75,20 @@ pub struct ConflictDetectionResponse {
     pub conflicts: Vec<Conflict>,
 }
 
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
+pub struct PacedTrainsConflicts {
```

Comment:
This designates a paced train occurence, not a conflict (it's only used in `Conflict` to designate an occurence). What do you think of renaming this `PacedTrainOccurrenceId` (it's basically the way to identify an occurrence of paced train)?